test/spec/bulk-memory.wast

```diff
@@ -165,3 +165,13 @@
 (invoke "drop_passive")
 (invoke "drop_passive")
 (invoke "drop_active")
+
+;; memory.size
+(module
+  (memory 1)
+  (func (export "size") (result i32)
+    (memory.size)
+  )
+)
+
+(assert_return (invoke "size") (i32.const 65536))
```

Comment:
I don't think this should be in `bulk-memory.wast` since it is not a bulk memory operation, but I'm ok with it if there is no better place.

Reply:
All our spec tests (except for the ones prefixed with `old_`) are in sync with the spec repo (may not now but as of a month ago or something), so I was kind of lazy to go to the spec repo and update the test.... Maybe I should at some point. Talked in person :)